invenio_communities/errors.py

```diff
@@ -26,3 +27,18 @@ def __init__(self, query_arguments):
                 )
             )
         )
+
+
+class LogoSizeLimitError(CommunityError):
+    """The provided logo size exceeds limit."""
+
+    def __init__(self, limit, file_size):
+        """Initialise error."""
+        super().__init__(
+            _(
+                "Logo size limit exceeded. Limit: {limit} MB Given: {file_size} MB".format(
+                    limit=ceil(limit/10**6),
```

Comment:
Same question as before. This conversion is being done in many places, perhaps there is a better way to deal with this? What do you think?